packages/shorebird_code_push_protocol/lib/src/models/shorebird_plan.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:shorebird_code_push_protocol/shorebird_code_push_protocol.dart';
+
+part 'shorebird_plan.g.dart';
+
+/// {@template shorebird_plan}
+/// A Shorebird plan.
+/// {@endtemplate}
+@JsonSerializable()
+class ShorebirdPlan {
```

Comment:
I can be convinced otherwise, but the idea behind this was to allow us to separate things the backend needs (stripe price id, for example) from things the client needs. The client doesn't (and shouldn't) have a notion of the StripeEnvironment, as another example.

Reply:
(this is the same thinking behind duplicating something like AppMetadata)

Reply:
Yeah I understand that I'm specifically wondering about the maxTeamSize vs collaboratorsEnabled

Reply:
`maxTeamSize` is more flexible than `collaboratorsEnabled`. Given our current messaging on shorebird.dev ("1 developer"), I can imagine a world where we introduce a new tier that supports teams of up to, say, 3.